lib/Sanger/CGP/CavemanPostProcessing/Flagger.pm

```diff
@@ -271,6 +294,26 @@ sub _tumFetch{
 	return 1;
 }
 
+sub _getDistanceFromGapInRead{
+  my ($cigar_array,$rdPosIndexOfInterest) = @_;
+  my $min_gap_dist = -1;
+  my $currentRp = 0;
+  foreach my $cigSect(@{$cigar_array}){
+    if($cigSect->[0] eq $MATCH_CIG || $cigSect->[0] eq $SKIP_CIG ||
+          $cigSect->[0] eq $INS_CIG || $cigSect->[0] eq $SOFT_CLIP_CIG){
+      $currentRp+=$cigSect->[1];
+    }elsif($cigSect->[0] eq $DEL_CIG){
+      my $dp_start = $currentRp+1;
+      my $dp_end = $currentRp+$cigSect->[1];
+      my $tmp_dist = max(abs($rdPosIndexOfInterest-$dp_start),abs($dp_end-$rdPosIndexOfInterest));
+      if($tmp_dist < $min_gap_dist || $min_gap_dist == -1){
+        $min_gap_dist = $tmp_dist;
+      }
+    }
+  }
+  return $min_gap_dist;
+}
+
```

Comment:
Duplication of code in `lib/Sanger/CGP/CavemanPostProcessing.pm`, is there something blocking moving this into a shared module?  It's already a function call so it needing to go to a different module isn't an overhead.

Reply:
Cleaned up code. Need to compare new and old and ensure same results then get rid of old code base as new one is far more flexible imho. Not for this feature though. 39ca2b5